geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStore.scala

```diff
@@ -9,31 +9,36 @@
 package org.locationtech.geomesa.blob.core
 
 import java.io.File
-import java.util
+import java.util.{Iterator => JIterator, Map => JMap}
 
+import com.google.common.collect.Maps
 import com.google.common.io.Files
 import com.typesafe.scalalogging.LazyLogging
 import org.apache.accumulo.core.data.{Key, Mutation, Range, Value}
-import org.apache.accumulo.core.security.Authorizations
 import org.apache.hadoop.io.Text
+import org.geotools.data.{DataStore, Query}
 import org.geotools.data.collection.ListFeatureCollection
 import org.geotools.data.simple.SimpleFeatureStore
-import org.geotools.data.{Query, Transaction}
+import org.geotools.filter.identity.FeatureIdImpl
 import org.locationtech.geomesa.accumulo.AccumuloVersion
 import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, _}
 import org.locationtech.geomesa.accumulo.util.{GeoMesaBatchWriterConfig, SelfClosingIterator}
 import org.locationtech.geomesa.blob.core.AccumuloBlobStore._
-import org.locationtech.geomesa.blob.core.handlers.BlobStoreFileHandler
+import org.locationtech.geomesa.blob.core.handlers.{BlobStoreFileHandler, _}
+import org.locationtech.geomesa.blob.core.interop.GeoMesaBlobStore
 import org.locationtech.geomesa.utils.filters.Filters
 import org.locationtech.geomesa.utils.geotools.Conversions._
 import org.locationtech.geomesa.utils.geotools.SftBuilder
+import org.opengis.feature.simple.SimpleFeature
 import org.opengis.filter.Filter
 
 import scala.collection.JavaConversions._
 import scala.util.control.NonFatal
 
-class AccumuloBlobStore(ds: AccumuloDataStore) extends LazyLogging with BlobStoreFileName {
+class AccumuloBlobStore(gds: DataStore) extends GeoMesaBlobStore
+  with BlobStoreFileName with LazyLogging {
 
+  private val ds = gds.asInstanceOf[AccumuloDataStore]
```

Comment:
was there any reason to make this a private val instead of leaving it as the constructor signature? I would change it back